debian/changelog

```diff
@@ -1,3 +1,9 @@
+opx-base-model (3.152.0+nmu1) unstable; urgency=medium
```

Comment:
package version in debian/changelog and confugure.ac should be same : '3.151.0+num1'